### ISSUE TITLE
IPN Notifications

### ISSUE THREAD

**Issue by guidodipaolo** [mercadopago/cart-woocommerce#3]
Hi !! 

I am having some problems with the IPN notificacions. 
I ahve configured the URL as the documentation indicates, but when I test it it appears the following error: 

`Respuesta: {"protocolVersion":{"major":1,"minor":1,"protocol":"HTTP"},"reasonPhrase":"Internal Server Error","statusCode":500}` 

Thanks !!

**Comment by marcelohama** [comment]
Hi, guidodipaolo

Thank you for your feedback.
Please, check v1.0.2 if it now correctly triggers your URL.

**Comment by guidodipaolo** [comment]
Thanks !! 

I have installed the 1.0.3 version and now the IPN notifications works great. 
Now i am having two problems: 
1) The woocommerce emails are not sending. If I change this plugin to the developed by Carlos Sanches, emails works. 
2) When the plugin redirects to Mercadopago it asks the shipping method. Why it happends ? There is a posibility to avoid shipping step ? 

Thanks

**Comment by JorgeAlan** [comment]
I agree is confusing for the costumers when asks the shipping method.

**Comment by guidodipaolo** [comment]
Any answer. I will have to change to an unoficial woocommerce plugin
